.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: objective-c
 cache: cocoapods
-osx_image: xcode8.3
+osx_image: xcode9.3
 before_install:
   - brew uninstall carthage
-  - HOMEBREW_NO_AUTO_UPDATE=1 brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/6ae4f69a652fb0ecb102b0c9216378679a4f1b92/Formula/carthage.rb # 0.22.0
+  - HOMEBREW_NO_AUTO_UPDATE=1 brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/45dd24d8dfa7a2fb69812c678ceb34be0c16e295/Formula/carthage.rb # 0.29.0
```

Comment:
This force upgrade should not be needed anymore. Could you please remove the `before_install:`?